src/esp/sim/Simulator.h

```diff
@@ -845,6 +845,29 @@ class Simulator {
       const assets::AssetInfo& assetInfo,
       const assets::RenderAssetInstanceCreationInfo& creation);
 
+#ifdef ESP_BUILD_WITH_VHACD
+  /**
+   * @brief Runs convex hull decomposition on a specified file. Creates an
+   * object attributes referencing a newly created convex hull asset, and
+   * returns the attribute's handle.
+   *
+   * @param filename The MeshMetaData filename to be converted.
+   * @param params VHACD params that specify resolution, vertices per convex
+   * hull, etc.
+   * @param renderCHD Specifies whether or not to render the coinvex hull asset,
+   * or to render the original render asset.
+   * @param saveCHDToObj Specifies whether or not to save the newly created
+   * convex hull asset to an obj file.
+   * @return The handle of the newly created object attributes.
+   */
+  std::string convexHullDecomposition(
+      const std::string& filename,
+      const assets::ResourceManager::VHACDParameters& params =
+          assets::ResourceManager::VHACDParameters(),
+      const bool renderCHD = false,
+      const bool saveCHDToObj = false);
```

Comment:
```suggestion
      bool renderCHD = false,
      bool saveCHDToObj = false);
```

No need to make values (as opposed to `&` references) `const` in a function declaration, has no effect and only adds noise to the generated documentation.

On the other hand, tn the *definition* in the cpp it's fine, and encouraged :)